### PR TITLE
Add function methods `torch.sum()`, and etc.

### PR DESCRIPTION
**Pull request by kaiidams**
- torch.argmax()
- torch.argmin()
- torch.argsort()
- torch.normal(double, double)
- torch.sum()

### CODE REVIEW

**Comment by GeorgeS2019** [comment]
## Math operations
### Pointwise Ops
- [x] Torch.Sum

File: src/TorchSharp/Tensor/Tensor.Factories.cs [hunk 1]
```diff
@@ -950,6 +950,22 @@ public static Tensor normal(Tensor mean, Tensor std, torch.Generator? generator
...
+        /// <param name="std">The standard deviation for all distributions</param>
+        /// <param name="size">A sequence of integers defining the shape of the output tensor.</param>
+        /// <param name="dtype"></param>
+        /// <param name="device"></param>
+        /// <param name="requires_grad"></param>
+        /// <param name="generator">An optional random number generator</param>
+        /// <returns></returns>
+        public static Tensor normal(double mean, double std, ReadOnlySpan<long> size, torch.ScalarType? dtype = null, torch.Device? device = null, bool requires_grad = false, torch.Generator? generator = null)
+        {
+            return randn(size, dtype: dtype, device: device, requiresGrad: requires_grad, generator: generator) * std + mean;
```
Comment (NiklasGustafsson) [inline src/TorchSharp/Tensor/Tensor.Factories.cs:966]: If device is 'null,' should it use the std.device instead of the default device, perhaps?

Comment (kaiidams) [reply src/TorchSharp/Tensor/Tensor.Factories.cs:966]: `torch.rand()` should handle the case where `device` is `null`. As the user may call it like `torch.rand(size, dtype: torch.float64)` without `device`. `torch.InitializeDevice(null)` falls back to `torch.CPU`. We may want to call C++ `torch::rand()` function without device option if we want to make sure it is compatible with PyTorch?

Comment (NiklasGustafsson) [reply src/TorchSharp/Tensor/Tensor.Factories.cs:966]: @kaiidams -- I was confused. I was thinking that std and mean were tensors, and that the device should follow them unless non-null. std and mean are, of course, doubles, so the default device makes sense.

**Review by NiklasGustafsson — APPROVED** [review]

**Pull request merged**